From a47b5167c4c1b55b2f51e29ab0782c2659bec312 Mon Sep 17 00:00:00 2001
From: Siarhei Siamashka <siarhei.siamashka@nokia.com>
Date: Tue, 08 Dec 2009 06:52:34 +0000
Subject: ARM: added '.arch armv7a' directive to NEON assembly file

This fix prevents build failure due to not accepting PLD instruction when
compiling for armv4 cpu with the relevant -mcpu/-march options set in CFLAGS.
---
diff --git a/pixman/pixman-arm-neon-asm.S b/pixman/pixman-arm-neon-asm.S
index e8ccf77..e66fb86 100644
--- a/pixman/pixman-arm-neon-asm.S
+++ b/pixman/pixman-arm-neon-asm.S
@@ -41,6 +41,7 @@
 
     .text
     .fpu neon
+    .arch armv7a
     .altmacro
 
 #include "pixman-arm-neon-asm.h"
--
cgit v0.8.3-6-g21f6
